### DISCUSSION TITLE
Config.tx Question

### DISCUSSION THREAD

**Discussion by EdRaponi** [lllyasviel/Fooocus#3380] [Q&A] [answered]
Is it possible to set the default values via config.txt for the following values:
1. Input Image - I would like to set this to true so it is checked when I start Fooocus
2. Negative Prompt - I would like to populate with a list of 10 values that I find myself typing over & over again

Picture attached
![Focus](https://github.com/user-attachments/assets/d024ffb2-42fb-41ec-9d92-282968e4e504)

**Comment by mashb1t** [comment, marked as answer]
1. see https://github.com/lllyasviel/Fooocus/discussions/3370#discussioncomment-10164201 (coming soon)
2. you can use the key `default_prompt_negative`, but i'd propose you copy the default template, rename it and change it there

**Reply by EdRaponi** [reply]
I already tried this before posting, if you set default_prompt_negative to "", it works fine; however, if you try to use a list of prompts enclosed in bracket and double-quotes ["prompt 1", "prompt 2", "prompt 3"] it does not work.  When the program is executed it states default_prompt_negative will be set to "".

**Reply by mashb1t** [reply]
default_prompt_negative is a string, so you can e.g. set it to `"default_prompt_negative": "prompt 1, prompt 2, prompt 3"`.

**Reply by EdRaponi** [reply]
Thank you!

**Reply by EdRaponi** [reply]
What about setting the default of Input Image as checked (true)?

**Reply by mashb1t** [reply]
@EdRaponi Did you read the linked discussion?

**Reply by EdRaponi** [reply]
Thank you - The first time I didn't completely follow the chain of notes you provided.

Works perfectly!

**Comment by lgwjames** [comment]
in my config file i have

`  "default_prompt_negative": "__neg__ ",`

this points to my wildcard

(((ugly)))), (((duplicate))), ((morbid)), ((mutilated)), out of frame, extra fingers, mutated hands, ((poorly drawn hands)), ((poorly drawn face)), (((mutation))), (((deformed))), ((ugly)), blurry, ((bad anatomy)), (((bad proportions))), ((extra limbs)), cloned face, (((disfigured))), out of frame, ugly, extra limbs, (bad anatomy), gross proportions, (malformed limbs), ((missing arms)), ((missing legs)), (((extra arms))), (((extra legs))), (fused fingers), (too many fingers), (((long neck))), (burry), ((burry)), cropped, deformed, dull, poor lighting, deformed iris, deformed pupils, cropped, out of frame, jpeg artifact,Image compression, Distorted, Grainy, Out of Focus, Blurry,DOF, Noisy, Watermark,Text, Copyright, low resolution, shaky, too bright
too dark
Poorly lit
Pixelated
Poor quality
low quality
Unclear
Blocked
Artifacts
Banding
Truncated
Out of Frame
disjointed
incoherent
asymmetry
disorganized
jumbled
tasteless
tacky

**Reply by EdRaponi** [reply]
This is awesome! Thank you for sharing!!
